litr/src/main/java/com/linkedin/android/litr/render/AudioRenderer.kt

```diff
@@ -26,9 +26,11 @@ private const val TAG = "AudioRenderer"
 
 class AudioRenderer @JvmOverloads constructor(
     private val encoder: Encoder,
-    private val filters: MutableList<BufferFilter> = mutableListOf()
+    filters: MutableList<BufferFilter>? = null
```

Comment:
Should this be `private val filters: List<BufferFilter> = listOf()`?

We're requiring that this list is mutable, but we're always just casting it as an immutable list.

Reply:
So we want the list to be immutable internally, but cannot accept an immutable list in constructor, because Java doesn't like it. Therefore, all this finagling.

Reply:
OK 👍  but would be curious to see examples where this causes issues in Java if you come across them in the future, as I haven't seen issues around this before